packages/snap-preact-components/src/components/Molecules/Slideout/Slideout.tsx

```diff
@@ -108,6 +105,29 @@ export function Slideout(properties: SlideoutProps): JSX.Element {
 	);
 }
 
+const ButtonContent = (props: { content: string | JSX.Element | undefined; toggleActive: () => void }): JSX.Element => {
+	const { content, toggleActive } = props;
+
+	if (content && typeof content == 'string') {
+		return (
+			<div
+				onClick={() => toggleActive()}
+				dangerouslySetInnerHTML={{
+					__html: content,
+				}}
+			/>
+		);
+	} else if (content && typeof content == 'object') {
+		let elem = () => {
+			let elemm: JSX.Element = cloneElement(content, {
+				onClick: () => toggleActive(),
+			});
+			return elemm;
+		};
+		return elem();
+	} else return <></>;
+};
+
```

Comment:
Haven't tested this but it's interesting that the prop type for `buttonContent` already included `JSX.Element`. Wonder if this was just never working 🤔

Reply:
It was working before, but it would always add the element inside of the wrapping ss__slideout__button div. 

You also could only pass inner text as a string and not html as a string, which wasn't totally necessary but I thought it made it easier to use if it just worked with anything.